src/styles/blocks/minute.scss

```diff
@@ -0,0 +1,8 @@
+.minute {
+  background-color: chartreuse;
+  width: $hans-width;
+  height: ($watch-size / 2) * 0.8;
```

Comment:
it's not clear what is 0.8 and 0.95 for the second class, will be better to create variables for these values